growth/too/templates/plan_manual.html

```diff
@@ -67,6 +67,39 @@
                     0 for standard imaging and readout
                     </small>
                 </div>
+                <div class="form-group col-sm-4">
+                    <label for=mode_num>Number of EWR Images (ZTF Only)</label>
```

Comment:
Suggest adding a tooltip to explain what this option does.

Reply:
Does the `<small>` text below work, or is there some other way that you implement that?

Reply:
I'd like to see in the help text an explanation of what EWR stands for and what it means.